src/Storages/MaterializedView/RefreshTask.cpp

```diff
@@ -294,7 +306,7 @@ void RefreshTask::refreshTask()
 
             try
             {
-                executeRefreshUnlocked(view);
+                executeRefreshUnlocked(append);
                 refreshed = true;
             }
             catch (...)
```

Comment:
Wdyt about logging about exponential backoff (possibly, with the delay duration) in addition to _"Refresh failed"_?